apache-tika.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: apache-tika
+  version: 3.0.0
+  epoch: 0
+  description: The Apache Tika toolkit detects and extracts metadata and text from over a thousand different file types (such as PPT, XLS, and PDF).
+  copyright:
+    - license: Apache-2.0
+
```

Comment:
What runtime dependencies do we need to be able to run `java -jar /tika-server-standard-3.0.0.jar`? They should be added here so that running `apk add apache-tika` will result in a working server, and so that the finaly image only needs the `apache-tika` and `apache-tika-compat` packages.